.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,40 @@
+name: CI
+
+on: [push]
```

Comment:
I'm afraid that the GitHub Actions could take to much time if we done them on every push. What you think of doing it only on develop and master?